embedding/embedlite/embedshared/EmbedLiteWindowChild.cpp

```diff
@@ -199,9 +196,44 @@ void EmbedLiteWindowChild::CreateWidget()
               );
   GetWidget()->UpdateBounds(true);
 
+  // Initialize ScreenManager
+  RefreshScreen();
+
   mInitialized = true;
   Unused << SendInitialized();
 }
 
+void EmbedLiteWindowChild::RefreshScreen()
+{
+  LayoutDeviceIntRect rect;
+  if (mRotation == ROTATION_0 || mRotation == ROTATION_180)
+    rect = mBounds;
+  else
+    rect = LayoutDeviceIntRect(0, 0, mBounds.Height(), mBounds.Width());
+
+  AutoTArray<RefPtr<Screen>, 1> screenList;
+  RefPtr<Screen> screen = new Screen(rect, rect, mDepth, mDepth, DesktopToLayoutDeviceScale(mDensity), CSSToLayoutDeviceScale(1.0f), GetWidget()->GetDPI());
```

Comment:
It looks to me that this GetDPI() is not the right one as this GetWidget() is nsWindow where as EmbedLitePuppetWidget::GetDPI reads back the dpi from the EmbedLiteViewChild.

I'd say that simpler to pass down dpi to here within SetScreenProperties